### PR TITLE
cli: display information about the CNFCERT run

### PR DESCRIPTION
**Pull request by jmontesi**
There are four sections displayed:

1. Basic information about the CNFCERT version and artifacts 
2. Live results for the tests that are running
3. Table that summarizes the result of each test suite 
4. Log output for those tests that failed

### CODE REVIEW

File: internal/cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	Green  = "\033[32m"
+	Yellow = "\033[33m"
+	Blue   = "\033[34m"
+	Purple = "\033[35m"
+	Cyan   = "\033[36m"
+	Gray   = "\033[37m"
+	White  = "\033[97m"
+)
+
+const banner = `
```
Comment (rdavid) [inline internal/cli/cli.go:19]: It looks fantastic! Could you please share how you created this banner? Additionally, I'd like to point out that there's a possibility that some fonts used might not be under the Apache license.

**Review by edcdavid — APPROVED** [review]

**Review by sebrandon1 — APPROVED** [review]

**Comment by sebrandon1** [comment]
@jmontesi In a subsequent PR, can you make one of those video/gifs of the test suite running with your new change here and put it on the README?  To show the [ PASS ], [ FAIL ], etc flying by?  It might look cool.

**Review by rdavid — APPROVED** [review]

File: internal/cli/cli.go (hunk 1 above)
Comment (jmontesi) [reply internal/cli/cli.go:22]: Comment added.

**Comment by jmontesi** [comment]
> @jmontesi In a subsequent PR, can you make one of those video/gifs of the test suite running with your new change here and put it on the README? To show the [ PASS ], [ FAIL ], etc flying by? It might look cool.

Sure, good idea.

**Review by greyerof — APPROVED** [review]

**Pull request merged**